lib/Net/Async/Blockchain/BTC.pm

```diff
@@ -285,6 +285,8 @@ async sub transform_transaction {
 
         my @categories       = keys %categories;
         my $transaction_type = scalar @categories > 1 ? 'internal' : $categories[0];
```

Comment:
Note for Reviewer:
ONly change is this & `t/btc.t` for test coverage. Rest is `make tidy`